presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergErrorCode.java

```diff
@@ -38,7 +38,8 @@ public enum IcebergErrorCode
     ICEBERG_INVALID_TABLE_TIMESTAMP(12, USER_ERROR),
     ICEBERG_ROLLBACK_ERROR(13, EXTERNAL),
     ICEBERG_INVALID_FORMAT_VERSION(14, USER_ERROR),
-    ICEBERG_UNKNOWN_MANIFEST_TYPE(15, EXTERNAL);
+    ICEBERG_UNKNOWN_MANIFEST_TYPE(15, EXTERNAL),
+    ICEBERG_CACHE_WRITE_ERROR(15, INTERNAL_ERROR);
```

Comment:
A duplicate code here, is this intentional?

Reply:
This was probably unintentional after fixing a merge conflict

Reply:
fixed